### PR TITLE
Document Browser settings configuration in Studio

### PR DESCRIPTION
**Pull request by fermujica**
This article will render as shown below:

preview

### CODE REVIEW

**Review by guidota — APPROVED** [review]

File: modules/ROOT/pages/browser-settings.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+= Configure Studio's Browser Settings
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio comes bundled with a default browser to display elements of the Anypoint Platform UI. For example, when you deploy to CloudHub from Studio, the Runtime Manager UI displayed in Studio is rendered using a built-in browser.
+
+It is recommended to use the built-in browser, however, if the default browser fails to display you can configure Studio to your system's native browser to render the Anypoint Platform UI:
```
Comment (kbostrom) [inline modules/ROOT/pages/browser-settings.adoc]:
```suggestion
This built-in browser is the default. However, if it fails to display, you can configure Studio to use your system's native browser to render the Anypoint Platform UI:
```

File: modules/ROOT/pages/browser-settings.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+. Navigate to *Preferences* > *Anypoint Studio* > *Browser*
```
Comment (kbostrom) [inline modules/ROOT/pages/browser-settings.adoc]: period
```suggestion
. Navigate to *Preferences* > *Anypoint Studio* > *Browser*.
```

File: modules/ROOT/pages/browser-settings.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+. Select the type of default browser for Studio:
++
+image::browser-configuration.png[]
+<1> *Built-in*: Uses the built-in browser. This is the recommended setting.
+<2> *Native*: Uses the native browser of your operative system.
```
Comment (kbostrom) [inline modules/ROOT/pages/browser-settings.adoc:15]: The text above the task steps states already that the default is the built-in browser and that you can configure the native alternative, so I think all of this should be reduced to something like...:

. Change the default browser selection of *Builtin* to *Native*.

File: modules/ROOT/pages/browser-settings.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+. Click *Preview* to test the browser's configuration. +
```
Comment (kbostrom) [inline modules/ROOT/pages/browser-settings.adoc]:
```suggestion
. Click *Preview* to verify the changed configuration. +
```

File: modules/ROOT/pages/browser-settings.adoc [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+If the page previews correctly, then Studio's browser is correctly configured.
```
Comment (kbostrom) [inline modules/ROOT/pages/browser-settings.adoc]: Do we need this sentence if we change the one above it?
Also, I can see a possibility that my suggestion could have subtly changed meaning (verifying the config change vs verifying the resulting display), so please use your best judgment.

**Review by kbostrom — APPROVED** [review]
Here you go!

**Pull request merged**